actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/TenantMigrationActor.java

```diff
@@ -349,6 +354,17 @@ private String validateOrgExternalIdOrOrgIdAndGetOrgId(
                   ResponseCode.invalidParameterValue.getErrorMessage(),
                   (String) migrateReq.get(JsonKey.ORG_EXTERNAL_ID),
                   JsonKey.ORG_EXTERNAL_ID));
+        } else {
+          // Fetch locationids of the suborg and update the location of sso user
+          OrganisationClient orgClient = new OrganisationClientImpl();
```

Comment:
Instead of making ES search call , better get org details by id.
or
Better instead of calling orgExternalService.getOrgIdFromOrgExternalIdAndProvider and just getting orgId, call
OrgServiceImpl.getInstance().esGetOrgByExternalId() and get the complete org details and update location details